playbooks/example-use/playbook.yml

```diff
@@ -34,6 +34,18 @@
           - pulp_settings.azure_location is defined
           - pulp_settings.default_file_storage is defined
       when: pulp_install_object_storage == "azure"
+
+    - name: Check if pulpcore repo exists
+      stat:
+        path: /etc/yum.repos.d/pulpcore.repo
+      register: pulpcorerepo
+
+    - name: Remove pulpcore fedorapeople
+      lineinfile:
+        path: /etc/yum.repos.d/pulpcore.repo
+        regexp: 'fedorapeople'
+        line: "baseurl = {{ pulp_pkg_repo }}"
+      when: pulpcorerepo.stat.exists
```

Comment:
Not sure where is the best place to put it,
I put it here just for confirming it would fix the test

Reply:
This seems like the best place.